storage/src/test/java/tech/pegasys/teku/storage/server/VersionedDatabaseFactoryTest.java

```diff
@@ -38,6 +41,9 @@ public class VersionedDatabaseFactoryTest {
   public static final int MAX_KNOWN_NODE_CACHE_SIZE = 10_000;
   private final Eth1Address eth1Address =
       Eth1Address.fromHexString("0x77f7bED277449F51505a4C54550B074030d989bC");
+
+  final AsyncRunnerFactory asyncRunnerFactory =
+      AsyncRunnerFactory.createDefault(new MetricTrackingExecutorFactory(new NoOpMetricsSystem()));
```

Comment:
I suspect we don't want an actual async runner factory here as it makes tests multithreaded.  Likely just want something that creates StubAsyncRunner instances.

Or if we don't need the actual factory, just create a StubAsyncRunner and use that.